internal/vulnstore/postgres/store.go

```diff
@@ -18,7 +19,9 @@ import (
 
 // store implements all interfaces in the vulnstore package
 type Store struct {
-	pool *pgxpool.Pool
+	pool        *pgxpool.Pool
+	initMu      sync.RWMutex
```

Comment:
can't this be dropped to just an atomic bool? its okay if more the one go routine writes "true" to it, nothing will flip it back to "false" and all writes of "true" are atomic to any viewers.

Reply:
That's not correct, because the calls may be concurrent. I think my latest change does it correctly, though.